app/editor/src/features/admin/av-overviews/OverviewSubscribers.tsx

```diff
@@ -0,0 +1,64 @@
+import { useFormikContext } from 'formik';
+import React from 'react';
+import { useUsers } from 'store/hooks/admin';
+import { FlexboxTable, ITableInternal, ITablePage, ITableSort, IUserModel } from 'tno-core';
+
+import { subscriberColumns } from './constants';
+import { IAVOverviewTemplateForm } from './interfaces';
+import { ListFilter } from './ListFilter';
+
+/**
+ * The page used to view and edit reports.
+ * @returns Component.
+ */
+export const OverviewSubscribers: React.FC = () => {
+  const { values, setFieldValue } = useFormikContext<IAVOverviewTemplateForm>();
+  const [{ users }, { findUsers }] = useUsers();
+
+  React.useEffect(() => {
+    async function getUsers() {
+      await findUsers({ page: 1, quantity: users.quantity });
+    }
+    getUsers();
```

Comment:
You have to place a simple catch handler or the console will throw errors `.catch(() => {})`

Reply:
Updated, thanks